pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -274,6 +281,7 @@ let
       "--with-gnu-as"
       "--without-gnu-ld"
     ]
+    ++ lib.optional (targetPlatform.libc == "mlibc") "--disable-libsanitizer"
```

Comment:
Could this be unified with the one for musl below?

Reply:
Oh I don't know what I was thinking when I said that. musl appears to be adding `--disable-libsanitizer` for all builds just like mlibc. That condition appears to only be for deduplicating the flag. This also made me notice that this flag is being passed twice on mlibc. I think I can remove them from `crossConfigureFlags` as `configureFlags` is applied to all builds? It would also get rid of the ambiguity with the uclibc comment.

Reply:
Yes, I think that's right.